api/bus.go

```diff
@@ -150,6 +150,14 @@ type ContractAcquireResponse struct {
 	LockID uint64 `json:"lockID"`
 }
 
+// ContractsPrunableDataResponse is the response type for the
+// /contracts/prunable endpoint.
+type ContractsPrunableDataResponse struct {
+	ContractSizes []ContractSize `json:"contractSizes"`
+	TotalPrunable uint64         `json:"totalPrunable"`
+	TotalSize     uint64         `json:"totalSize"`
```

Comment:
Why not make the totals methods on the response type? To reduce the redundant info.

Reply:
I still think about our API a lot from the context of a user consuming it with `curl`. I see what you are saying though but in this case the whole reason of the PR was to have an endpoint I can `curl` that gives me two aggregated fields and a sorted breakdown.

How strongly do you feel about it? I would usually just implement the remark but then I'd have to manually sum the totals or write small scripts for it to debug pruning. The bloat is negligible right

Reply:
I updated the other thing but would argue we can keep this as-is. It's two extra fields saving you the trouble of doing it yourself.